doc/nrf/releases_and_maturity/migration/migration_guide_2.9.0-nRF54H20-1.rst

```diff
@@ -165,3 +167,15 @@ nrfutil suit
        nrfutil install suit=0.9.0 --force
 
     For more information, consult the `nRF Util`_ documentation.
+
+nrf-regtool
```

Comment:
@jonathannilsen  please confirm the regtool entry

Reply:
The version is correct.

Just curious about the instruction below this, I don't use the toolchain manager but I thought it would include all the python requirements that are in requirements.txt. Is it necessary to manually upgrade the pip dependencies that have been bumped since the last release?

Reply:
@kenr Could you clarify this?